.github/workflows/e2e-stress-test-flake-fix.yml

```diff
@@ -44,8 +54,8 @@ jobs:
     name: Stress test E2E flake fix
     env:
       DISPLAY: ""
-      QA_DB_ENABLED: true
-      CYPRESS_QA_DB_MONGO: true
```

Comment:
Previously this was hard-coded to `true` and when this happen, we would skip making postgres snapshots.

https://github.com/metabase/metabase/blob/d965ab224746709557ddbd369095c9c759c6ca2e/e2e/snapshot-creators/qa-db.cy.snap.js#L16-L22

Reply:
This was on me 🤦